src/node_version.h

```diff
@@ -51,4 +51,4 @@
  */
 #define NODE_MODULE_VERSION 48 /* Node.js v6.0.0 */
 
-#endif  /* SRC_NODE_VERSION_H_ */
+#endif  // SRC_NODE_VERSION_H_
```

Comment:
heh. this rule was the first thing I spent time fixing on my initial attempt. wish I had taken the time to see how many lines of code were affected first.